src/frontend/app/features/service-catalog/services.service.ts

```diff
@@ -102,8 +103,22 @@ export class ServicesService {
       true
     ).entities$;
   }
+  private getServiceInstances = () => {
+    const paginationKey = createEntityRelationPaginationKey(serviceInstancesSchemaKey, this.cfGuid);
```

Comment:
There's some very similar code in specify-details-step.component initServiceInstances. Would be good to this function there